### PR TITLE
Basic naive bmm schedule

### PR DESCRIPTION
**Pull request by zasdfgbnm**
This PR modifies `scheduleMatmul` to support basic bmm by parallelizing the batch dim on BIDz. I don't know how good is the perf for this schedule, but this is not the purpose of this PR. The main purpose for this PR is to have something functionally working, so that I can unblock my next step for adding a bmm scheduler to the registry which also dispatch to `scheduleMatmul` for the separate kernel split-k. I will leave it to @drzejan2 to implement a performant bmm.

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
!build

**Comment by drzejan2** [comment]
Changes look good to me, thanks @zasdfgbnm for clean changes that introduced bgemm support!

**Review by drzejan2 — APPROVED** [review]

**Comment by drzejan2** [comment]
!build

File: csrc/scheduler/matmul.cpp [hunk 1]
```diff
@@ -734,22 +746,25 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
 
   //  0   1  2  3   4   5   6  7  8  9  10
```
[the remark this answers is not in the document]
Comment (zasdfgbnm) [reply csrc/scheduler/matmul.cpp]: Thanks for catching! Updated.

**Pull request merged**